packages/dashboard/src/features/database/hooks/useBackupConfig.ts

```
@@ -0,0 +1,63 @@
+    error,
+    refetch,
+  } = useQuery({
+    queryKey: BACKUP_CONFIG_QUERY_KEY,
+    queryFn: () => backupService.getBackupConfig(),
+    enabled: !isCloudHostingMode,
+    staleTime: 2 * 60 * 1000,
+  });
+
+  const updateBackupConfigMutation = useMutation({
+    mutationFn: (patch: UpdateDatabaseBackupConfig) =>
+      backupService.updateBackupConfig(patch),
```

ESLint

[error] Delete `⏎·····`

(prettier/prettier)